packages/web-components/fast-foundation/src/skeleton/README.md

```diff
@@ -24,9 +37,27 @@ _Example:_
 ></fast-skeleton>
 ```
 
+### Pattern
+
+A URL for an image asset may be passed to the `pattern` attribute. In this mode, the `fast-skeleton` component is used as a container for a transparent SVG that may express a more complex placeholder
+
+```html
+<fast-skeleton
+    style="
+        --neutral-fill-rest: #e1dfdd;
```

Comment:
Do we need to add the `--neutral-fill-rest` definition here? We don't generally recommend setting CSS custom properties manually that are controlled by DesignToken.

Reply:
Not sure. If I recall, I took that directly from the fixtures.

Reply:
I'm going to remove it. It's not present in explorer.